commons-okhttp/src/main/scala/com/avsystem/commons/okhttp/rest/RestClient.scala

```diff
@@ -0,0 +1,65 @@
+package com.avsystem.commons
+package okhttp.rest
+
+import java.io.IOException
+
+import com.avsystem.commons.annotation.explicitGenerics
+import com.avsystem.commons.rest.RawRest.HandleRequest
+import com.avsystem.commons.rest.{HeaderValue, HttpBody, RawRest, RestMetadata, RestResponse}
+import okhttp3.{Call, Callback, HttpUrl, MediaType, OkHttpClient, Request, RequestBody, Response}
+
+object RestClient {
+
+  def apply[@explicitGenerics Real: RawRest.AsRealRpc : RestMetadata]
+  (client: OkHttpClient, scheme: String, host: String, port: Int):Real = RawRest.fromHandleRequest[Real](asHandleRequest(client, scheme, host, port))
```

Comment:
Can we just pass `baseUrl` instead of separate scheme, host and port? What if the base URL also needs a base path?